libs/component-tests/src/main/java/org/codice/ddf/test/component/configurators/DdfOptions.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.test.component.configurators;
+
+import static org.ops4j.pax.exam.CoreOptions.bundle;
+import static org.ops4j.pax.exam.CoreOptions.composite;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.editConfigurationFilePut;
+
+import org.codice.ddf.test.component.configurators.BundleOptionBuilder.BundleOption;
+import org.codice.ddf.test.component.configurators.FeatureOptionBuilder.FeatureOption;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.util.PathUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DdfOptions implements ApplicationOptions {
+
+    public static final String HTTP_PORT_KEY = "org.codice.ddf.system.httpPort";
+
+    public static final String HTTPS_PORT_KEY = "org.codice.ddf.system.httpsPort";
+
+    public static final String ROOT_CONTEXT_KEY = "org.codice.ddf.system.rootContext";
+
+    public static final String ROOT_CONTEXT = "/services";
+
+    public static final String SERVLET_CONTEXT_KEY = "org.apache.cxf.servlet.context";
+
+    public static final String OSGI_SERVICE_HTTP_PORT_KEY = "org.osgi.service.http.port";
+
+    public static final String OSGI_SERVICE_HTTP_PORT_SECURE_KEY =
+            "org.osgi.service.http.port.secure";
+
+    public static final String SYSTEM_HOSTNAME_KEY = "org.codice.ddf.system.hostname";
+
+    private static final String SYSTEM_PORT_KEY = "org.codice.ddf.system.port";
+
+    private static final String SYSTEM_PROTOCOL_KEY = "org.codice.ddf.system.protocol";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DdfOptions.class);
+
+    private final PortFinder portFinder;
+
+    public DdfOptions(PortFinder portFinder) {
+        this.portFinder = portFinder;
+    }
+
+    @Override
+    public Option get() {
+        return composite(//
```

Comment:
Are this comment and one on line 81 here on purpose?

Reply:
Removed those since we've changed formatter since then.